shared/src/libs/helpers/sanitize-input/sanitize-input.helper.ts

```diff
@@ -0,0 +1,17 @@
+import sanitizeHtml, { type IOptions } from 'sanitize-html';
+
+const sanitizeInput = (input: string, options?: IOptions): string => {
+  const charactersToRemove = /[&<>]/g;
+  const cleanedInput = input.replaceAll(charactersToRemove, '');
```

Comment:
won't sanitizeHtml remove all html characters?
```suggestion
```

Reply:
It doesn't remove it, just changes it for example "<" to ";" and "&" to "amp;". So what should I do with this?

Reply:
it's ok if it will change tags to mnemonic, that's what we need